java/consumer/build.gradle

```diff
@@ -15,21 +15,19 @@
 
 apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
-apply plugin: 'kotlin-android-extensions'
```

Comment:
Hi Shawn, just want to confirm the app compiles fine after removing this? I suspect yes, but want to make sure.